### PR TITLE
refactor: type update any to unknown

### PR DESCRIPTION
**Pull request by Inder782**
Relates to #3790 

### Description:

Tried to refactor the code by changing some types from `any` to `unknown`. I made these changes incrementally to get feedback on whether I am moving in the right direction. I'll proceed or adjust my approach based on the feedback received.

Refactor

### Reviewer Actions

- [ ] I have manually tested the changes in the preview environment.
- [x] I have reviewed the code.
- [x] I understand that "request changes" will block this PR from merging.

### CODE REVIEW

**Comment by ZeRego** [comment]
@Inder782 this is in the right direction 👍 linting seems to be failing. There are `fix-format` and `fix-linting` commands to help with this.

**Review by ZeRego — CHANGES_REQUESTED** [review]
Needs to pass build/test check

**Comment by Inder782** [comment]
Thanks for the feedback, @ZeRego. I'll use fix-formatting and fix-linting, and then I'll also refactor the entire frontend folder for the first pull request. After that, I'll move forward.

File: packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts [hunk 1]
```diff
@@ -1636,7 +1638,7 @@ const useEchartsCartesianConfig = (
                     validCartesianConfig?.eChartsConfig.grid,
                 ),
             },
-            // If using shared chart colors, we don't specify any colors at the top level,
+            // If using shared chart colors, we don't specify unknown colors at the top level,
```
Comment (ZeRego) [inline packages/frontend/src/hooks/echarts/useEchartsCartesianConfig.ts:1641]:
```suggestion
            // If using shared chart colors, we don't specify any colors at the top level,
```

**Comment by ZeRego** [comment]
Closing this PR, feel free to reopen when fixes are ready 👍

**Pull request closed without merging**